appendices/common-arguments.md

```diff
@@ -0,0 +1,131 @@
+# Common Arguments
+
+Here is the list and explanation of several arguments, which can be used with most commands.
+
+## if
+
+The `if` argument followed by a condition, defines whether a command will be executed or skipped. The condition is a C# macro and if the expression contain some spaces, we advise you to paste it within `⊂⊃` for a clearer readability.
```

Comment:
if the expression contains*
and it's not for a clearer readability but because G1ANT throws exception if it sees a condition like this:

`♥x == 3`

instead of this:

`♥x==3`

Reply:
```suggestion
The `if` argument followed by a condition, defines whether a command will be executed or skipped. The condition is a C# macro and if the expression contains some spaces, you have to embrace it with `⊂⊃` special characters.
```